app/App.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
+import ReactRouter from 'react-router';
 import Header from './Header';
 
 const App = React.createClass({
   render() {
-    return(
-      <div>
-        <Header />
-        {this.props.children}
-      </div>
-    );
+    const path - this.props.location.pathname;
```

Comment:
Yes, sorry.  I just fixed it and pushed the correction to my branch.

On Thu, Feb 25, 2016 at 7:16 PM, Gordon Buhler <notifications@github.com>
wrote:

> In app/App.js
> <https://github.com/HypnoSheep/HypnoSheep/pull/81#discussion_r54192796>:
>
> >  import Header from './Header';
> >
> >  const App = React.createClass({
> >    render() {
> > -    return(
> > -      <div>
> > -        <Header />
> > -        {this.props.children}
> > -      </div>
> > -    );
> > +    const path - this.props.location.pathname;
>
> Should this be
> const path = this.props.location.pathname; ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HypnoSheep/HypnoSheep/pull/81/files#r54192796>.
>